metrics/github/cli.py

```diff
@@ -15,9 +15,10 @@
 log = structlog.get_logger()
 
 
-def open_prs(prs, org, days_threshold):
+def old_prs(prs, org, days_threshold):
```

Comment:
There are more instances of "open" that should be "old" below. I can't comment on them individually because GitHub has strong but incorrect opinions about which lines are relevant to this PR. Please can you search through this code for instances of "open" and kill almost all of them?

Reply:
Done in 13132df